### PR TITLE
Content update for the NUC desktop install page

### PR DESCRIPTION
**Pull request by caldav**
## Done

- New NUC image
- Adjusted content

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/download/iot/intel-nuc-desktop](http://0.0.0.0:8001/download/iot/intel-nuc-desktop)
- Ensure the content is accurate and Ubuntu image links are working

### CODE REVIEW

**Comment by caldav** [comment]
This is now ready for review and a bit urgent.

File: templates/download/iot/intel-nuc-desktop.html [hunk 1]
```diff
@@ -17,21 +17,17 @@ <h1>Install Ubuntu Desktop on the Intel<sup>&reg;</sup> NUC</h1>
...
-            <li class="p-list__item is-ticked">An Ubuntu SSO account with an SSH key</li>
             <li class="p-list__item is-ticked">An Intel NUC with BIOS updated to the latest version (<a class="p-link--external" href="http://www.intel.com/content/www/us/en/support/boards-and-kits/000005850.html">update instructions</a>)</li>
-            <li class="p-list__item is-ticked">1 USB 2.0 or 3.0 flash drive (2GB minimum)</li>
-            <li class="p-list__item is-ticked">A monitor with an HDMI interface</li>
-            <li class="p-list__item is-ticked">A Mini HDMI to HDMI cable</li>
+            <li class="p-list__item is-ticked">1 USB 2.0 or 3.0 flash drive (4GB minimum for Dawson Canyon NUCs, 2GB for older generations)</li>
             <li class="p-list__item is-ticked">A USB keyboard and a mouse</li>
-            <li class="p-list__item is-ticked">A monitor with VGA or HDMI interface</li>
+            <li class="p-list__item is-ticked">A monitor with an HDMI interface</li>
             <li class="p-list__item is-ticked">A VGA or HDMI cable</li>
```
Comment (sparkiegeek) [inline templates/download/iot/intel-nuc-desktop.html]: a VGA cable isn't going to help you if you succeed with previous point and have a monitor with an HDMI port

Comment (caldav) [reply templates/download/iot/intel-nuc-desktop.html]: Thanks for catching it

**Comment by pmahnke** [comment]
@caldav I updated the [copy doc](https://docs.google.com/document/d/1hDkrBqpJXWaENYZMSxd8P-XLFQ2LQqS1wNe4o2MGLho/edit#heading=h.3ewmb38urwx7) for you.

One small change, then you are good to go.

**Comment by caldav** [comment]
@pmahnke what's the change?

**Pull request merged**